### ISSUE TITLE
Stage-damage Functions dialog edit top table to display "Write Details to..."

### ISSUE THREAD

**Issue by jbslaughter** [HydrologicEngineeringCenter/HEC-FDA#793]
Build created 2/21/23

Create New Stage-damage function dialog box modify bottom of top table to fully display checkbox text option to write structure stage damage details to a csv file...

![CreateNew_Stage-DamageFunction_WriteStructureDetail](https://user-images.githubusercontent.com/89167848/221225965-f3537bd4-1b2c-4c18-945b-e2b4aad55d8e.png)

**Comment by jbslaughter** [comment]
I realized expanding the window showed the full text "Write Details to CSV" - this isn't imperative, but could be fixed by setting the left side of the Configure and Compute panel to keep the table from covering up the "Write Details to CSV" text.

**Comment by rnugent3** [comment]
That's right - expanding makes just about everything easier on the eyes. I'll include some text about that in the user interface section. 

We'll keep this issue live for a future enhancement. Thanks for pointing it out, @jbslaughter

**Comment by Brennan1994** [comment]
reworked
